ELE_Patcher/Utility/ModSupport.cs

```diff
@@ -421,8 +421,8 @@ private static Dictionary<string, FormLink<IImageSpaceGetter>> GenerateCustomIma
 			#region Legacy of the Dragonborn
 			if (mods.Contains(KeyLotDb))
 			{
-				// Check version
-				bool old = !LegacyoftheDragonborn.Cell.DBMDGArmoryWest.TryResolve(state.LinkCache, out _);
+                // Check version
+                bool old = !LegacyoftheDragonborn.Cell.DBMDGArmoryWest.TryResolve(state.LinkCache, out var rec);
```

Comment:
Not sure why the `out _` was making the compiler unhappy.  Added `out var rec` for now, but also could do a
`bool old = !LegacyoftheDragonborn.Cell.DBMDGArmoryWest.TryResolve(state.LinkCache) == null;`
I suppose